components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesBookmarksStorage.kt

```diff
@@ -192,4 +192,8 @@ open class PlacesBookmarksStorage(context: Context) : PlacesStorage(context), Bo
     override fun getHandle(): Long {
         return places.getHandle()
     }
+
+    override fun registerWithSyncManager() {
+        throw NotImplementedError("Use getHandle instead")
```

Comment:
nit: Is this a TODO for us? If so, can we file a follow-up and link to the ticket here?

Reply:
Yes and no. We will, hopefully, one day move away from using `getHandle` entirely, but that totally depends on all of the storage layers exposing `registerWithSyncManager` in A-S. I'll file a ticket.

Reply:
https://github.com/mozilla-mobile/android-components/issues/10128